src/basic/points.js

```diff
@@ -114,10 +118,10 @@ Point.getDraw = function(c3ss, id) {
 
 	if (checkMarkerSym(c3ss)) {
 
-		Object.assign(
-				point,
-				getColors(c3ss),
-				getWidths(c3ss)
+		point = mergeWith(
```

Comment:
It is intended for having a decision function for conflicts...

Here both, `getColors` and `getWidths` return and object with an equal property wich others, this property is outline, so if outline already exists I use merge again to have a deep merge.

With `Object.assing` we have the latest outline, not a merged outline.